R/metab.bookkeep.R

```diff
@@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
 		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
 		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
-		freq <- nobs
+		freq <- nobs - 1
```

Comment:
This is an interesting discussion, and I've maybe swapped sides to the midnight to midnight side. If we are going to change metab, then we might want to consider additional fitting options to increase functionality, such as having an option to fit from sunrise to sunrise or fit parameters over different time windows (e.g. multiple days).